Pod/Classes/ViewControllers/ARMapContainerViewController.m

```diff
@@ -30,15 +30,15 @@ @interface ARMapContainerViewController () <PulleyDelegate, PulleyDrawerViewCont
 @end
 
 static UIScrollView *
-FindFirstVerticalScrollView(UIView *view)
+FindFirstScrollView(UIView *view)
 {
     for (UIView *subview in view.subviews) {
-        if ([subview isKindOfClass:UIScrollView.class] && ([(UIScrollView *)subview contentSize].height > subview.frame.size.height)) {
```

Comment:
The issue here was that the outermost scroll view has a wide content size, at least at first. Since it contains the rest of the scroll views (now), this check wasn't necessary anymore.